dask/dataframe/groupby.py

```diff
@@ -1681,6 +1681,14 @@ def get_group(self, key):
 
     @_aggregate_docstring()
     def aggregate(self, arg, split_every=None, split_out=1, shuffle=None):
+        split_out = 1 if split_out is None else split_out
```

Comment:
Given the logic below, it looks like we want to drop this line 

```suggestion
```

Reply:
Whoops, getting careless